doc/Cellpose-StarDist-Segmentation.ipynb

```diff
@@ -433,23 +433,21 @@
    "cell_type": "code",
    "execution_count": 12,
    "metadata": {},
-   "outputs": [
-    {
-     "ename": "",
-     "evalue": "",
-     "output_type": "error",
-     "traceback": [
-      "\u001b[1;31mThe Kernel crashed while executing code in the the current cell or a previous cell. Please review the code in the cell(s) to identify a possible cause of the failure. Click <a href='https://aka.ms/vscodeJupyterKernelCrash'>here</a> for more info. View Jupyter <a href='command:jupyter.viewOutput'>log</a> for further details."
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
     "# convert to ImageJ ROI in RoiManager\n",
     "ij.ui().show(data)\n",
     "convert.index_img_to_roi_manager(ij, nuc_labels)\n",
     "convert.index_img_to_roi_manager(ij, cyto_labels[0])"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Note:** You may have to adjust the brightness/contrast on the displayed image to view both the nucleus and cytoplasmic signals."
```

Comment:
Shall we add code to the notebook to set the brightness/contrast appropriately, so that manual tuning afterward  becomes unnecessary?

Reply:
Agreed, I added a few lines to the last cell that selects the second channel and applies B&Cs auto adjust to it. I removed the note. f205852cdf0968b3667c2a8f2402aa19fced6a71